src/UI/Settings/NetImport/Options/NetImportOptionsView.js

```diff
@@ -32,7 +32,7 @@ var view = Marionette.ItemView.extend({
 		var refresh=params.get('refresh');
 		if (oauth && refresh){
 			//var callback_url = window.location.href;
-			history.pushState('object', 'title', (window.location.href).replace(window.location.search, ''));
+			history.pushState('object', 'title', (window.location.href).replace(window.location.search, '')); // jshint ignore:line
```

Comment:
Only way to clean up this warning is to disable the rule for this line due to history being global in JS and not needing to be defined. I'm not familiar with jsHint so I just disabled linting for the whole line. If someone wants to disable just the `is-not-defined` rule specifically for this line, that would be fine.